docs/getting-started/connect-atomist-to-ci.md

```diff
@@ -64,6 +63,358 @@ In the Notification configuration section, configure the endpoint URL as `https:
+
+If you run a [Multibranch Pipeline job][multibranch], your build is 
+described in a [Jenkinsfile][jenkinfile]. The integration with Atomist is 
+done within the Jenkinsfile using the [post][jenkinsfilepost] section. A
+multibranch pipeline build job is interesting because it allows you to 
+concurrently build all the branches of your project.
+
+If you do not have the plugin installed, ask your Jenkins administrator to add
+the [Multibranch Pipeline project][multibranch] plugin.
+
+<div class="ss-container">
+  <img src="../images/jenkins-multibranch-pipeline-plugin.png" alt="Create Multibranch Pipeline Plugin" class="ss-medium">
+</div>
+
+By default, the Multibranch pipeline plugin relies on anonymous calls to the 
+GitHub API to scan existing repositories and their branches. However, the number
+of anonymous calls that can be made from a given IP is 
+[rather limited][ghratelimit], it is therefore best if you set the 
+credentials for an account that will perform those queries.
+
+The plugin supports various credential providers such as username/password or 
+private keys. Below is the description on using a username/password approach 
+where the password is a GitHub personal access token.
+
+Go to your [GitHub settings][ghpat] to generate a new token with only the 
+`repo` scope:
+
+<div class="ss-container">
+  <img src="../images/github-jenkins-token.png" alt="GitHub personal access token for Jenkins" class="ss-medium">
+</div>
+
+Then, ask your Jenkins administrator to create cerdentials for your GitHub
+username and token:
+
+<div class="ss-container">
+  <img src="../images/jenkins-credentials-system.png" alt="Jenkins credentials" class="ss-medium">
+</div>
+
+<div class="ss-container">
+  <img src="../images/jenkins-creds-for-github.png" alt="Jenkins add new GitHub credentials" class="ss-medium">
+</div>
+
+Now, you can create a multibranch pipeline build job as usual:
+
+<div class="ss-container">
+  <img src="../images/jenkins-create-multibranch-pipeline-job.png" alt="Create Multibranch Pipeline Job" class="ss-medium">
+</div>
+
+A multibranch pipeline project expects at least one source where to monitor
+and build from. This source is essentially your project on GitHub and it must
+have a file named `Jenkinsfile` at the top-level directory. That file contains
+the description of your build pipeline.
+
+<div class="ss-container">
+  <img src="../images/jenkins-multibranch-pipeline-source.png" alt="Configure Multibranch source" class="ss-medium">
+</div>
+
+Set the `Owner` to the GitHub organisation holding your project. The plugin will
+automatically scan for repositories in that org. 
+
+Select that new pair of credentials in the Multibranch build job so that it uses
+that account to scan for repositories and branches in the repository you will
+be building.
+
+Once a repository has been selected, attach the appropriate credentials for the
+checkout. You may leave the other settings to their default values or tune as
```

Comment:
`Once a repository has been selected, attach the appropriate credentials used for access.`